src/index.js

```diff
@@ -7,12 +7,17 @@
 import { resolve } from 'path';
 import Config from 'webpack-config';
 
-export default new Config().extend(
+const config = new Config().extend(
```

Comment:
Please revert all edits to this file as I already applied this change to the `production` config in #21